### ISSUE TITLE
BUG : If used inside singlechildscrollview it fails to load the list

### ISSUE THREAD

**Issue by shukill** [OutdatedGuy/firebase_pagination#1]
```dart
SingleChildScrollView(
  child: FirestorePagination(
    query: firebaseServices.postsQuery(false),
    onEmpty: const Center(
      child: Padding(
        padding: EdgeInsets.all(20),
        child: Text('You are all catched up for the day'),
      ),
    ),
    bottomLoader: const SizedBox(
      width: 80,
      height: 80,
      child: Center(
        child: CircularProgressIndicator(
          color: primaryColor,
        ),
      ),
    ),
    initialLoader: const Center(
      child: CircularProgressIndicator(
        color: primaryColor,
      ),
    ),
    limit: 2,
    isLive: true,
    viewType: ViewType.wrap,
    shrinkWrap: true,
    itemBuilder: (context, documentSnapshot, index) {
      Post post = Post.fromSnap(documentSnapshot);
      return KeyedSubtree(
        key: ValueKey(post.postId),
        child: PostCard(
          snap: post,
          user: widget.user,
        ),
      );
    },
  ),
),
```

**Comment by OutdatedGuy** [comment]
Yo @shukill, thanks for using this package.

Can you share some screenshots with and without using `SingleChildScrollView`.
I think the problem is mostly related to [this](https://stackoverflow.com/questions/56131101/how-to-place-a-listview-inside-a-singlechildscrollview-but-prevent-them-from-scr).

Also, 
```dart
viewType: ViewType.wrap
``` 
uses `SingleChildScrollView` as parent. So need for an extra `SingleChildScrollView` on top of `FirestorePagination`.

**Comment by shukill** [comment]
Hey @OutdatedGuy so sorry for the late reply, it's working fine without them, I pinned point the issue, the scroll controller with the package never engages as it is wrapped up by a controller already, in this case, we can optionally pass as controller to the package also,

**Comment by OutdatedGuy** [comment]
So a property `controller` would do it right?

I'll work on it.

**Comment by OutdatedGuy** [comment]
Fixed in [72b9067](https://github.com/OutdatedGuy/firebase_pagination/commit/72b90670916c10d0126c60a3871395f6bb9700bc)

**Comment by shukill** [comment]
Perfect :)
